### PR TITLE
Add help text + request signing section header to IdP form

### PR DESCRIPTION
**Pull request by david-crespo**
Following up on https://github.com/oxidecomputer/console/pull/2511#issuecomment-2460418920. Let's be sooooooo helpful.

image

image

### CODE REVIEW

**Comment by david-crespo** [comment]
Moved on top of #2510 so we can get extra snazzy.

image

File: app/forms/idp/create.tsx [hunk 1]
```diff
@@ -120,7 +122,36 @@ export function CreateIdpSideModalForm() {
...
+        name="name"
+        control={form.control}
+        description={
+          <>
+            A short name for the provider in our system. Users will see it in the path to
+            the login page:{' '}
+            <code>
+              /login/{silo}/saml/{name.trim() || 'idp-name'}
+            </code>
+            .
```
Comment (charliepark) [inline app/forms/idp/create.tsx:151]: If we're ending the copy with `<code>`, I think it looks better (and clearer) without the ultimate period. Thoughts about leaving that off?

Comment (david-crespo) [reply app/forms/idp/create.tsx:151]: I have it in there because it’s two sentences. Otherwise I would leave it out.

Comment (david-crespo) [reply app/forms/idp/create.tsx:151]: Hm, yeah I think you’re right. I’ll take it out.

**Pull request merged**